tests/views/test.del-pay-method-confirmation.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import TestUtils from 'react/lib/ReactTestUtils';
+
+import $ from 'jquery';
+import ConfirmDelPayMethod from 'views/management/confirm-del-pay-method';
+
+import * as helpers from '../helpers';
+
+describe('Confirm Delete Pay Methods', function() {
+
+  beforeEach(function() {
+
+    this.fakePayMethods = [
+      {
+        'id': 1,
+        'resource_uri': '/pay-method/1/',
+        'truncated_id': '4321',
+        'type_name': 'visa',
+      }, {
+        'id': 2,
+        'resource_uri': '/pay-method/2/',
+        'truncated_id': '1234',
+        'type_name': 'amex',
+      }, {
+        'id': 23,
+        'resource_uri': '/pay-method/3/',
+        'truncated_id': '4321',
+        'type_name': 'mastercard',
+      },
+    ];
+
+    this.fakeSubs = [
+      {
+        'id': 1,
+        'paymethod': '/pay-method/2/',
+        'resource_uri': '/sub/1/',
+        'seller_product': {
+          'external_id': 'mozilla-concrete-mortar',
+          'public_id': 'mozilla-concrete-mortar',
+          'resource_pk': 1,
+          'resource_uri': '/generic/product/1/',
+        },
+      }, {
+        'id': 2,
+        'paymethod': '/pay-method/2/',
+        'resource_uri': '/sub/2/',
+        'seller_product': {
+          'external_id': 'mozilla-concrete-brick',
```

Comment:
you can probably leave off `external_id` because I don't think we use it anywhere in the whole app